### PR TITLE
Add updates to cloud networks faq

### PR DESCRIPTION
**Pull request by kylelaffoon**
Relates to #1413

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-168c2d2110/how-to/)

File: content/cloud-networks/cloud-networks-faq.md [hunk 1]
```diff
@@ -112,13 +116,21 @@ We are launching security groups as limited availability in all data centers, so
 
 With the limited availability launch in early 2015, we support only inbound security groups on both PublicNet and ServiceNet interfaces. This means that customers can filter incoming traffic to their PublicNet and ServiceNet ports. We will add outbound security group support later in 2015.
```
Comment (catlook) [inline content/cloud-networks/cloud-networks-faq.md]: outbound is now supported

File: content/cloud-networks/cloud-networks-faq.md [hunk 2]
```diff
@@ -112,13 +116,21 @@ We are launching security groups as limited availability in all data centers, so
...
+A security group can't be added as a child of another security group. You also can't edit a security group rule - you must add a new security group to replace the old one.
+
 #### Will security groups be supported via the neutron client?
 
 Yes. Users can provision security groups via the neutron client.
 
 #### Is this functionality integrated with and available from the Cloud Control Panel?
 
-Not yet. The product will be available soon. In the interim, you can use either the neutron client or the API.
+Yes, but only inbound PublicNet and ServiceNet security groups are currently available in the control panel. To access additional features, you can use either the neutron client or the API.
```
Comment (catlook) [inline content/cloud-networks/cloud-networks-faq.md]: is this true?   or is it a hold over from before we supported outbound at all?

**Comment by catlook** [comment]
only a couple of questions about outbound functions.  see inline comments.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7408d04f90/how-to/)

File: content/cloud-networks/cloud-networks-faq.md (hunk 2 above)
Comment (kylelaffoon) [inline content/cloud-networks/cloud-networks-faq.md]: @trentgillaspie Are only inbound security groups available in the control panel?

Comment (trentgillaspie) [inline content/cloud-networks/cloud-networks-faq.md]: Currently, yes. Only inbound security groups are available in the control panel, and only for public and service net.

Thank you,
Trent

Trent Gillaspie

Rackspace | Product Manager | 303.386.6115

Strategic | Futuristic | Communication | Responsibility | Arranger

/////////////////////////////////////////////////////

**Comment by trentgillaspie** [comment]
Getting lost in the merges, but doesn't look like this was updated on the site yet. Correct? Any outstanding items needed? Thanks!

**Pull request merged**